js/weird-string-case.test.js

```diff
@@ -0,0 +1,9 @@
+const toWeirdCase = require('./weird-string-case');
+
+describe('weird string case', () => {
+
+  test('Got string with spaces should return in weird case', () => {
```

Comment:
The test's full name will be:

> weird string case Got string with spaces should return in weird case

Does not look like a proper sentence 😄 Please adjust it